### PR TITLE
Bump node-disk-manager to 0.3.2

### PR DESCRIPTION
**Pull request by weihanglo**
**Problem:**
Disk auto-provisioning feature

**Solution:**
Bump harvester-node-disk-manager to 0.3.2

**Related Issue:**
https://github.com/harvester/harvester/issues/1241
https://github.com/harvester/harvester/issues/1599

**Test plan:**
See #1241

### CODE REVIEW

File: deploy/charts/harvester/Chart.lock [hunk 1]
```diff
@@ -7,10 +7,10 @@ dependencies:
   version: 0.1.0
 - name: harvester-network-controller
   repository: https://charts.harvesterhci.io
-  version: 0.2.1
+  version: 0.2.4
```
Comment (guangbochen) [inline deploy/charts/harvester/Chart.lock]: can we please help to reset this value, it will be bumped in another PR, thanks.

Comment (weihanglo) [reply deploy/charts/harvester/Chart.lock]: Fixed. Thanks!

**Review by guangbochen — APPROVED** [review]
LGTM
